tv_apps/smart-home/test/unit/mock_spatial_navigator.js

```diff
@@ -0,0 +1,22 @@
+ /* globals evt */
+
+(function(exports) {
+  'use strict';
+
+  var MockSpatialNavigator = function() {
+  };
+
+  MockSpatialNavigator.prototype = evt({
+    init: function() {},
+    focus: function() {},
+    //remove: function() {},
```

Comment:
nit: I think we should avoid comment code here. We could always add it back when we need it.